### PR TITLE
revise Backend Engineer JD

### PR DESCRIPTION
**Pull request by nodestory**

### CODE REVIEW

File: Backend Engineer.md [hunk 1]
```diff
@@ -1,38 +1,64 @@
...
+- Cloud services (AWS, GCP, DigitalOcean)
 - Strong English communication skills
-- Strong coding skills and computer science fundamentals
-- Experience with Golang
-- Knowledge of cryptography and cryptographic tools (Eg: Tor, Bitcoin) is a major plus
-- Knowledge of databases (PostgreSQL, LevelDB, Redis, …)
-- Knowledge of communication methods to interact with  applications and other services (WebSockets, RPC, JSON, …)
-- Knowledge of configuration management (Git, containers, Ansible, Helm, …) in a development environment
-- Able to take the initiative
+
```
Comment (nodestory) [inline Backend Engineer.md:55]: I'm thinking if we should add more descriptions about our culture.

File: Backend Engineer.md [hunk 2]
```diff
@@ -1,38 +1,64 @@
...
+- Maintain [the Bitmark blockchain](https://github.com/bitmark-inc/bitmarkd) and [the Bitmark Golang SDK](https://docs.bitmark.com)
+- Work with communities to advance [the Bitmark Protocol](https://bitmark.com/products/bitmark-protocol/faq).
+  The research topics include but are not limited to:
+    - [Decentralized Identifiers (DIDs)](https://www.w3.org/TR/did-core)
+    - [InterNFT](https://internft.org)
+    - [RGB](https://rgb-org.github.io)
+    - [Cosmos](https://cosmos.network)
 
 ## Profiles that are important to us
```
Comment (nodestory) [inline Backend Engineer.md:45]: I'm thinking if we should add more descriptions about our culture to this section `Profiles that are important to us`.

File: Backend Engineer.md [hunk 3]
```diff
@@ -1,38 +1,64 @@
...
+- Develop [the Autonomy pod](https://hackmd.io/CrNGQQSsTmKzo2YYogRqtA?view#Pod-Architecture), the Bitcoin full node designed for running in the cloud privately for each user  
+  - Increase the privacy, safety, and security of Autonomy pod, which will require [Torgap](https://github.com/BlockchainCommons/torgap/blob/master/README.md), [Signal messaging](https://signal.org/docs/), etc.
+  - Design a scalable architecture for running more Autonomy pods
+  - Lower the cost of running an Autonomy pod
+- Develop the backend of Autonomy app and other Bitcoin-related microservices powered by Bitmark (e.g. fee estimation, price history, explorer, etc.)
+- Work with communities to make the solutions [Gordian-certified](https://github.com/BlockchainCommons/Gordian)
+
+### Bitmark Protocol
+
+> In the digital age, the ability to set rules around the usage and exchange of assets has become increasingly difficult for individuals. Crypto rights are programmable rules to control assets. They are cryptographic, so they can be exchanged across the Internet. Bitmark is a blockchain-based, end-to-end protocol for crypto rights.
```
Comment (moskovich) [inline Backend Engineer.md]: Can you change this one to read:

"Bitmark Protocol is a system for organizing information about the world’s digital resources. People can activate — use, transfer, generate revenue from — resources safely with each other by verifying what a resource is and who controls it through Bitmark Protocol."

File: Backend Engineer.md (hunk 2 above)
Comment (moskovich) [reply Backend Engineer.md:45]: I don't think it's necessary. They should be able to understand our culture through our actions.

**Comment by moskovich** [comment]
Comments on job descriptions.

**Review by moskovich — CHANGES_REQUESTED** [review]
Try this.

**Pull request merged**